### ISSUE TITLE
Type error when used with Bluebird

### ISSUE THREAD

**Issue by vajahath** [scopsy/await-to-js#10]
I'm using [sequelizejs](http://docs.sequelizejs.com/)
```ts
	[err, dbResponse] = await to(
		user.findOne({
			where: {
				email: req.body.email,
				passwordHash: req.body.password,
			},
		}),
	);
```
err:
> Argument of type 'Bluebird<{}>' is not assignable to parameter of type 'Promise<{}>'.
  Property '[Symbol.toStringTag]' is missing in type 'Bluebird<{}>'.

**Comment by dygufa** [comment]
I had this problem too. Currently await-to-js only supports "real Promises", but async / await accepts any object with .then and .catch defined. 
This could be solved by changing the type description to accept  any returned object in witch .then and .catch are defined, I guess.
@scopsy, do you agree?

**Comment by scopsy** [comment]
Should be fixed by #21  I'll upload a new version to NPM asap. 
Sorry for not being so responsive.
